.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "Traffic dev env",
+	"image": "ghcr.io/xoolive/traffic/dev:latest",
+	// "build": {
+	// 	"dockerfile": "Dockerfile",
+	// 	"context": ".."
+	// },
+
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
```

Comment:
We could match this with the content of `https://github.com/xoolive/traffic/blob/master/.vscode/extensions.json`, but then I am not sure if that one would be ignored here

Reply:
I did that. Unfortunately, VS Code does not provide the means to just have one file. For now, we have to cope with the duplicate.